### ISSUE TITLE
IE11 not working - Syntax error in resizer.js?

### ISSUE THREAD

**Issue by wimlibby** [davidjbradshaw/iframe-resizer#352]
Pages not re-sizing in IE11 and console show syntax error in iframeresizer script. Works in other browsers, so don't know if this is IE 11 issue or not.

Below is the error log.  Note there are only three calls to iFrameSizer - I am used to seeing a lot more calls.  

**Note reference to syntax error:** It is referring to line 626 in iFrameResizer:
	`		iframe.contentWindow.postMessage( msgId + msg, target );`

[iFrameSizer][Host page: FrontendMainMenu_m1IframeOverlay] IFrame scrolling disabled for FrontendMainMenu_m1IframeOverlay

[iFrameSizer][Host page: FrontendMainMenu_m1IframeOverlay] [init] Sending msg to iframe[FrontendMainMenu_m1IframeOverlay] (FrontendMainMenu_m1IframeOverlay:8:false:true:32:true:true:null:lowestElement:null:null:0:false:parent:scroll) targetOrigin: javascript:'';

SCRIPT5022: SyntaxError
iframeResizer.min.js (8,6268)

SCRIPT438: Object doesn't support property or method 'autocomplete'
custom.asp (166,5)

[iFrameSizer][Host page: FrontendMainMenu_m1IframeOverlay] [iFrame requested init] Sending msg to iframe[FrontendMainMenu_m1IframeOverlay] (FrontendMainMenu_m1IframeOverlay:8:false:true:32:true:true:null:lowestElement:null:null:0:false:parent:scroll) targetOrigin: javascript:'';

SCRIPT5022: SyntaxError
iframeResizer.js (626,4)

Example page showing not working IE11 - works in other browsers.
[http://www.gapsychiatry.org/general/custom.asp?page=21]

**Comment by davidjbradshaw** [comment]
Nothing wrong with that line, I'd suggest fixing your bug on line 166 and see if that fixes things for you.

If you think it is an error in this project then please create a simple test case.

**Comment by wimlibby** [comment]
Update:

I turned off the targetOrigin requirement and it started to work in some of the IE11 pages where it wouldn't before - but not all.  I tried this because in the initial call it was sending targetOrigin: javascript: and not targetOrigin: domain.  No idea why this is happening one some pages in IE 11 only.

On the pages in IE 11 where it is still not working, when I turn the debugger on it now works. However, If I have the debugger off, it still fails.  

Here is example page that works in IE 11 with debugger on, but fails with it off:
[http://www.vapa.org/exhibitor/exhibitor.php](url)

is there a way to force a re-size once the last element on the page is loaded?

**Comment by davidjbradshaw** [comment]
Your page has a few JS errors on it, I expect this is causing side effects and breaking other things. Fix those errors and then see if this problem goes away.

**Comment by fjwon** [comment]
i found your sample can't work in ie 11 now.

**Comment by ChristianDavis** [comment]
I'm getting that same syntax error with Edge, does anyone have a cause or solution to the problem? I don't see any reason for the syntax error TBH. I think IE is taking issue with the logger?

**Comment by ChristianDavis** [comment]
checkOrigin: false,

If you have frames loading from a different domain or subdomain, Iframe Resizer does an origin check, 
https://github.com/davidjbradshaw/iframe-resizer#checkorigin . IE was wrapping the DOM exception into a Syntax error for some reason, causing the whole script to fail. In my case there was a timing hole from when I set document.domain, so it was a little tricky to catch. But I set checkOrigin to false and it works. I've since added the correct domains to the checkOrigin array. 

@fjwon
